Okta.Xamarin/Okta.Xamarin/OidcClient.cs

```diff
@@ -337,9 +364,15 @@ protected virtual string GetBasePath()
             return $"{Config?.OktaDomain}/oauth2/v1";
         }
 
-        protected virtual string GetAuthorizationServerBasePath(string authorizationServerId = "default")
+        protected virtual string GetAuthorizationServerBasePath()
         {
-            return $"{Config?.OktaDomain}/oauth2/{authorizationServerId}/v1";
+            string domain = Config?.OktaDomain;
+            if ((bool)domain?.EndsWith("/"))
+            {
+                domain = domain.Substring(0, domain.Length - 1);
+            }
+
+            return $"{domain}/oauth2/{AuthorizationServerId}/v1";
```

Comment:
Your base URL (aka issuer), is different depending on your authorization server. I suggest using the same logic we have in other SDKs, for example, [here](https://github.com/okta/okta-aspnet/blob/master/Okta.AspNet.Abstractions/UrlHelper.cs#L14).
For more info about Okta's authorization servers, check out [here](https://developer.okta.com/docs/concepts/auth-servers/).